thirdeye/thirdeye-pinot/src/main/resources/assets/javascript/HashService.js

```diff
@@ -42,11 +42,19 @@ HashService.prototype = {
     switch (key) {
       case HASH_PARAMS.ANOMALIES_START_DATE:
       case HASH_PARAMS.ANOMALIES_END_DATE:
+      case HASH_PARAMS.ANALYSIS_CURRENT_START:
+      case HASH_PARAMS.ANALYSIS_CURRENT_END:
+      case HASH_PARAMS.ANALYSIS_BASELINE_START:
+      case HASH_PARAMS.ANALYSIS_BASELINE_END:
         value = moment(value);
         break;
       case HASH_PARAMS.ANOMALIES_PAGE_NUMBER:
         value = Number(value);
         break;
+      case HASH_PARAMS.ANALYSIS_FILTERS:
+        if (typeof value === "string") {
```

Comment:
Seen, in that case since you have complete control over this input value, maybe a comment here indicating that, just incase another dev works on this codebase and might think this is a bug on first pass